src/data/nat/parity.lean

```diff
@@ -138,6 +138,18 @@ begin
   exact @modeq.modeq_mul _ _ 1 _ 1 h₁ h₂
 end
 
+theorem odd_mul : odd (m * n) ↔ odd m ∧ odd n :=
+by simp [not_or_distrib] with parity_simps
+
+theorem even.mul (hm : even m) (n) : even (m * n) :=
+even_mul.mpr $ or.inl hm
+
+theorem mul_even (m) (hn : even n) : even (m * n) :=
+even_mul.mpr $ or.inr hn
+
+theorem odd.mul_odd (hm : odd m) (hn : odd n) : odd (m * n) :=
+odd_mul.mpr ⟨hm, hn⟩
```

Comment:
tempting to shorten the name, since I can't think of anything else you'd want to use the short name for:
```suggestion
theorem odd.mul (hm : odd m) (hn : odd n) : odd (m * n) :=
odd_mul.mpr ⟨hm, hn⟩
```

Reply:
I'm fine with the shorter name if other people are, too.

Reply:
I guess it's fine. But we'll want `odd.mul_even` as well.

Reply:
Ooh, I see that `even.mul_right` makes more sense for that.

Reply:
Exactly...that is the change I effected in the "simplification" commit. I kept the name of this lemma as `odd.mul_odd` for that extra bit of clarity, but I'm not averse to `odd.mul` if people feel that name is sufficiently descriptive.